pyproject.toml

```diff
@@ -33,6 +33,7 @@ dependencies = [
     "importlib_metadata<=6.5.0; python_version<'3.8'",
     "legacy-cgi>=2.0.0; python_version>='3.13.0'",
     "opentelemetry-api>=1",
+    "packaging>=24.0",
```

Comment:
TODO: update this to the oldest version that works for our needs.

Reply:
we might want to figure out how to test the behavior with the older version in `riotfile.py`. I am worried about adding `"packaging": ["16.0", latest]` since it'll double the size of the tracer test suite, which is already massive, but I am just calling out that we can/should validate this compatibility in the test suite if possible (mostly if someone tries to use something from `packaging` which was added recently, CI might pass, but user apps might fail)

Reply:
Done, I moved tracer monkey tests to their own suite and added the version range for `packaging`.